bcda/cli.go

```diff
@@ -45,21 +46,79 @@ type cclfFileMetadata struct {
 	filePath  string
 }
 
-func importCCLF8(fileMetadata cclfFileMetadata) error {
-	if fileMetadata.filePath == "" {
-		return errors.New("file path (--file) must be provided")
+type cclfFileValidator struct {
+	totalRecordCount int
+	maxRecordLength  int
+}
+
+func importCCLF0(fileMetadata cclfFileMetadata) (map[string]cclfFileValidator, error) {
+	if (cclfFileMetadata{}) == fileMetadata {
+		err := errors.New("File CCLF0 not found")
+		log.Error(err)
+		return nil, err
+	}
+
+	file, err := os.Open(filepath.Clean(fileMetadata.filePath))
+	if err != nil {
+		log.Error(err)
+		return nil, err
+	}
+
+	log.Infof("File contains %s data for ACO %s at %s.\n", fileMetadata.env, fileMetadata.acoID, fileMetadata.timestamp)
```

Comment:
The details about environment, ACO, and timestamp were mostly for verification when this was a PoC, so we probably don't need all that now. With `importCCLFDirectory()` logging when the import is done or failed, maybe we could just say here that the CCLF0 file is being read, with the filename.